base_layer/wallet/Cargo.toml

```diff
@@ -55,6 +55,6 @@ env_logger = "0.7.1"
 prost = "0.8.0"
 
 [features]
-default=["bundled_sqlite"]
+c_integration = []
```

Comment:
So instead of just reverting these changes lets try and clean up the feature situation. As far as I can tell this c_integration flag doesn't do anything anymore so lets remove it.

Having Sqlite bundled by default seems correct to me so I would say leave this in and then we make sure that when the wallet_ffi build happens it doesn't use these features.